src/test/webapp/WEB-INF/spring/hazelcast-spring.xml

```diff
@@ -42,10 +42,6 @@
             <param-name>shutdown-on-destroy</param-name>
             <param-value>true</param-value>
         </init-param>
-        <init-param>
-            <param-name>client-config-location</param-name>
```

Comment:
If P2P mode is used, `client-config-location` should not be present and vice versa for client mode. We now throw errors if there's an extra unused property (`client-config-location`) in this case.